06.understand_sentiment/README.md

```diff
@@ -118,212 +121,208 @@ As alluded to in section [Model Overview](#model-overview), here we provide the
-
-# create trainer
-trainer = paddle.trainer.SGD(cost=cost,
-                                parameters=parameters,
-                                update_equation=adam_optimizer)
-```
+Notice for loading and reading IMDB data, it could take up to 1 minute. Please be patient.
 
-### Training
+```python
 
-`paddle.dataset.imdb.train()` will yield records during each pass, after shuffling, a batch input is generated for training.
+print("Loading IMDB word dict....")
+word_dict = paddle.dataset.imdb.word_dict()
 
-```python
+print ("Reading training data....")
 train_reader = paddle.batch(
     paddle.reader.shuffle(
-        lambda: paddle.dataset.imdb.train(word_dict), buf_size=1000),
-    batch_size=100)
+        paddle.dataset.imdb.train(word_dict), buf_size=25000),
+    batch_size=BATCH_SIZE)
+```
 
-test_reader = paddle.batch(
-    lambda: paddle.dataset.imdb.test(word_dict), batch_size=100)
+
+### Create Trainer
+
+Create a trainer that takes `train_program` as input and specify optimizer function.
+
+```python
+trainer = fluid.Trainer(
+    train_func=partial(train_program, word_dict),
+    place=place,
+    optimizer_func=optimizer_func)
 ```
 
-`feeding` is devoted to specifying the correspondence between each yield record and `paddle.layer.data`. For instance, the first column of data generated by `paddle.dataset.imdb.train()` corresponds to `word` feature.
+### Feeding Data
+
+`feed_order` is devoted to specifying the correspondence between each yield record and `paddle.layer.data`. For instance, the first column of data generated by `imdb.train` corresponds to `words`.
 
 ```python
-feeding = {'word': 0, 'label': 1}
+feed_order = ['words', 'label']
 ```
 
-Callback function `event_handler` will be invoked to track training progress when a pre-defined event happens.
+### Event Handler
+
+Callback function `event_handler` will be called during training when a pre-defined event happens.
+For example, we can check the cost by `trainer.test` when `EndStepEvent` occurs
 
 ```python
+# Specify the directory path to save the parameters
+params_dirname = "understand_sentiment_conv.inference.model"
+
 def event_handler(event):
-    if isinstance(event, paddle.event.EndIteration):
-        if event.batch_id % 100 == 0:
-            print "\nPass %d, Batch %d, Cost %f, %s" % (
-                event.pass_id, event.batch_id, event.cost, event.metrics)
-        else:
-            sys.stdout.write('.')
-            sys.stdout.flush()
-    if isinstance(event, paddle.event.EndPass):
-        with open('./params_pass_%d.tar' % event.pass_id, 'w') as f:
-            trainer.save_parameter_to_tar(f)
-
-        result = trainer.test(reader=test_reader, feeding=feeding)
-        print "\nTest with Pass %d, %s" % (event.pass_id, result.metrics)
+    if isinstance(event, fluid.EndStepEvent):
+        print("Step {0}, Epoch {1} Metrics {2}".format(
+                event.step, event.epoch, map(np.array, event.metrics)))
+
+        if event.step == 10:
```

Comment:
This is basically saying we will only train 10 steps. 
There is no need to call `trainer.stop()` since this is not a test.

Reply:
this model will run like for 200steps and couple couples with 1 epoch, I want to stop it fast so that user can see inference result

Reply:
In train.py, it will continue to run

Reply:
How long does it take to run? If the training takes about 10 minutes or so, we can let the users know the time cost.